src/state/GlobalState.tsx

```diff
@@ -249,33 +251,42 @@ export class GlobalState extends PureComponent<Props> {
   }
 
   async componentDidMount() {
-    const { i18n } = this.props
+    const { i18n, t } = this.props
+    const { gameUpdates, libraryStatus } = this.state
+
+    // Deals launching from protocol. Also checks if the game is already running
```

Comment:
Makes sense, but why was the original logic deleted?

Reply:
Dont need anymore since the error is being treated on the backend